README.md

```diff
@@ -56,3 +57,34 @@ And then run the image using
 
 ```docker container run  -d -p 3000:3000 laa-apply-for-legalaid-api```
 
+
+## Postgres Sql
+
+If you already have an installation of postgres you might want to use it.
+For development mode i am going for the simplest setup without setting any specific roles or users
+
+If you dont have postgres installed then you can run the following command to get up and running quickly
+
+```docker run  -d -v pg-data:/var/lib/postgresql/data -p 5432:5432  --name postgres  postgres:10.5```
+ 
+ the 10.5 version appears to be the latest and most likely the one we will be using in RDS.
```

Comment:
FYI You can see a full list of supported versions here https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.DBVersions

Reply:
Interesting they don't have 10.5 there.  I wonder why that is